collector/collector_test.go

```diff
@@ -41,6 +41,9 @@ func readMetric(m prometheus.Metric) MetricResult {
 	if pb.Counter != nil {
 		return MetricResult{labels: labels, value: pb.GetCounter().GetValue(), metricType: dto.MetricType_COUNTER}
 	}
+	if pb.Summary != nil {
+		return MetricResult{labels: labels, value: pb.GetSummary().GetSampleSum(), metricType: dto.MetricType_SUMMARY}
+	}
```

Comment:
Ideally would add quantiles as well, but that seems too much refactoring for this PR.